node/pkg/p2p/run_params.go

```diff
@@ -95,14 +98,22 @@ func NewRunParams(
 	return p, nil
 }
 
-// WithSignedObservationListener is used to set the channel to receive `SignedObservation“ messages.
+// WithSignedObservationListener is used to set the channel to receive `SignedObservation` messages.
```

Comment:
Since you corrected this one, did you notice all the others only have a beginning tick and not an ending tick?